www_src/components/select/select.less

```diff
@@ -0,0 +1,12 @@
+.select {
+  -webkit-appearance: none;
```

Comment:
unfortunately autoprefixer doesn't autoprefix "appearance" beacause it's technically non standard